### DISCUSSION TITLE
VRR/G-Sync compatible?

### DISCUSSION THREAD

**Discussion by slychoober1** [ClassicOldSong/Apollo#92] [Ideas]
This may seem like a useless feature at first, but I believe it can help with latency/smoothness when it comes to capping frame rate under the monitor refresh rate so vsync does not get engaged. This way I can keep the virtual display at 120hz but cap specific games to a target refresh rate ensuring there's no tearing/vsync input lag. Curious to know if this is feasible or not, other than that absolutely loving this fork!

**Comment by ClassicOldSong** [comment]
It is useless at all means on the server side, the more important part about frame latency is on the client side.

Assuming that you have a very good internet connection, the thing you need is get your client to stream at 120hz, then the actual latency can drop. Even if your client can't display at 120hz, it'll still benefit from the lower buffer time **on your client's renderer**, and potentially more smooth picture due to less displayed frame skips.
